graylog2-server/src/main/java/org/graylog2/filters/ExtractorFilter.java

```diff
@@ -70,34 +83,43 @@ public boolean filter(Message msg) {
         return false;
     }
 
-    private List<Extractor> loadExtractors(final String inputId) {
+    @Subscribe
+    public void handleInputCreate(final InputCreated event) {
```

Comment:
Adding `@SuppressWarnings("unused")` would disable the compiler warnings (and make the IDE happy).